tests/load/locustfiles/load.py

```diff
@@ -0,0 +1,70 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""Load test shape module."""
+import math
+from typing import Type
+
+import numpy
+from locust import LoadTestShape, User
+from locustfile import AutopushUser
+
+TickTuple = tuple[int, float, list[Type[User]]]
+
+
+class QuadraticTrend:
+    """A class that defines a quadratic LoadTestShape trend."""
+
+    a: float
```

Comment:
Could we use more descriptive variable names here?

Reply:
I'm willing to change them, but actually am at a loss for what to name them, since that's how they appear in the quadratic function, do you have a suggestion? I thought to add a comment, is that better?

Reply:
That's fair. 
I've [seen them referred](https://en.wikipedia.org/wiki/Quadratic_equation) to as the Quadradic, Linear and Constant coefficients, but those may not really help define them much to someone going through the code.
We can keep them _a_, _b_, and _c_, for now.